### PR TITLE
fix: builds with skip client

### PR DESCRIPTION
**Pull request by yogurtandjam**
This is a quick patch to get builds consistently working.

The skip team has released a version of their client that shaves off ~7MB from the bundle size (new bundle size is around ~44MB instead of ~51MB). 
This improvement along with an increase in the node heap to 6GB is enough to get builds working.

**This is NOT the end of the journey**. Skip is working on rebuilding some functionality in-house to reduce the bundle size by up to 6MB more 😮 . Once those changes are in we'll probably be able to remove the node heap size override.

**Why 6GB?**
The vercel container has 8GB of memory, <2 of which is taken by the container itself (running OS/infra). This is [in line with vercel's recommendation](https://vercel.com/guides/troubleshooting-sigkill-out-of-memory-errors#4.-increase-memory-allocation-for-node).

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -4300,6 +4280,16 @@ packages:
       utility-types: 3.10.0
     dev: false
 
+  /@keplr-wallet/unit@0.12.143(starknet@6.11.0):
```
Comment (tyleroooo) [inline pnpm-lock.yaml:4283]: lots of new dupes in here :(

**Comment by tyleroooo** [comment]
main chunk is 23,957.98 kB, which isn't terrible

**Comment by yogurtandjam** [comment]
> main chunk is 23,957.98 kB, which isn't terrible, but i guess it is +50% which is sad

yeah it's still a big increase but they're working on another patch that will further decrease the bundle size by ~6MB so it's getting better 👍

**Review by tyleroooo — APPROVED** [review]

**Comment by tyleroooo** [comment]
still seeing build ooms, gonna close and wait for a hot second

**Pull request closed without merging**